spec/proto/runtime/v1/appcallback.pb.go

```diff
@@ -8,14 +8,15 @@ package runtime
 
 import (
 	context "context"
+	reflect "reflect"
+	sync "sync"
+
```

Comment:
A little problem here: these `.pb.go` files are auto-generated by tools and will break the linter rules everytime it's re-generated. 
So, can we configurate the linter to ignore them?